main/context_processors.py

```diff
@@ -0,0 +1,21 @@
+from daycare.models import DayCare, Image
+from dogowner.models import DogOwner
+
+
+def navbar_extras(request):
+    navbar_picture_url = navbar_name = None
+    if request.user.is_authenticated:
+        if DogOwner.objects.filter(user=request.user).exists():
+            dog_owner = DogOwner.objects.filter(user=request.user).first()
+            navbar_picture_url = dog_owner.dog_picture_url
+            navbar_name = dog_owner.first_name + ' ' + dog_owner.last_name
```

Comment:
you can use the dogowner str functio https://github.com/redhat-beyond/Hotails/blob/b0145b1e6afc5940b6f6ae5be86ecf10ba0d5144/dogowner/models.py#L28

Reply:
Changed it.